test/kvec_test.cc

```diff
@@ -54,7 +54,7 @@ int main()
 	for (i = 0; i < M; ++i) {
 		std::vector<int> array;
 		array.reserve(N);
-		for (j = 0; j < N; ++j) array[j] = j;
+		for (j = 0; j < N; ++j) array.push_back(j);
```

Comment:
It seems that `push_back` here is useless, because `array.reserve` already reserves the space.